js/apps/admin-ui/src/identity-providers/ManageOrderDialog.tsx

```diff
@@ -57,11 +54,24 @@ export const ManageOrderDialog = ({
     setOrder(providerOrder);
   };
 
+  useFetch(
+    async () => adminClient.identityProviders.find(),
```

Comment:
No need for this to be `async`, there is already a Promise returned.
```suggestion
    () => adminClient.identityProviders.find(),
```